examples/dem/3d-rotating-drum/rotating-drum.prm

```diff
@@ -53,12 +54,14 @@ subsection lagrangian physical properties
     set young modulus particles           = 1e7
     set poisson ratio particles           = 0.24
     set restitution coefficient particles = 0.97
-    set friction coefficient particles    = 0.3
+    set friction coefficient particles    = 0.85
+    set rolling friction particles        = 0.025
   end
   set young modulus wall           = 1e7
   set poisson ratio wall           = 0.24
   set restitution coefficient wall = 0.85
-  set friction coefficient wall    = 0.35
+  set friction coefficient wall    = 0.85
+  set rolling friction wall        = 0.025
```

Comment:
That's a very weird set of parameters. Very high friction and low rolling friction? Are you sure?

Reply:
These parameters give this results, that I'm sure:


![image](https://github.com/user-attachments/assets/254981b3-8d10-43cf-83c6-dba98d464958)